// @alexa-games/sfb-f/dist/extensions/ACEExtension.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.isImporterExtension = exports.isDriverExtension = exports.isInstructionExtension = void 0;
function isInstructionExtension(extension) {
    return true; // Since "InstructionExtension" doesn't have required methods, any object is "technically" an InstructionExtension
}
exports.isInstructionExtension = isInstructionExtension;
function isDriverExtension(extension) {
    return extension.pre !== undefined;
}
exports.isDriverExtension = isDriverExtension;
function isImporterExtension(extension) {
    return extension.extendImportedContent !== undefined;
}
exports.isImporterExtension = isImporterExtension;
var GlobalDirectionsExtension_1 = require("./coreExtensions/GlobalDirectionsExtension");
Object.defineProperty(exports, "GlobalDirectionsExtension", { enumerable: true, get: function () { return GlobalDirectionsExtension_1.GlobalDirectionsExtension; } });
var LocalizationExtension_1 = require("./coreExtensions/LocalizationExtension");
Object.defineProperty(exports, "LocalizationExtension", { enumerable: true, get: function () { return LocalizationExtension_1.LocalizationExtension; } });
var SnippetExtension_1 = require("./coreExtensions/SnippetExtension");
Object.defineProperty(exports, "SnippetExtension", { enumerable: true, get: function () { return SnippetExtension_1.SnippetExtension; } });
var VoiceOverExtension_1 = require("./coreExtensions/VoiceOverExtension");
Object.defineProperty(exports, "VoiceOverExtension", { enumerable: true, get: function () { return VoiceOverExtension_1.VoiceOverExtension; } });
